src/components/Mutate/Task.tsx

```diff
@@ -69,41 +71,49 @@ export const MutateTask = ({ id }: { id?: string }) => {
   });
 
   const submitMutation = useQueryMutationWithClient((client, data) => {
+    if (!context?.settings) {
+      return Promise.resolve();
+    }
+
     return isEditMode
-      ? editTask(client, id, data)
-      : createTask(client, watch("folder"), data); //change
+      ? editTask(client, id, data, context?.settings)
+      : createTask(client, watch("folder"), data, context?.settings); //change
   });
 
   const taskQuery = useQueryWithClient(
     ["task", id as string],
-    (client) => getTaskById(client, id as string),
-    {
-      enabled: !!isEditMode,
-    }
+    (client) => getTaskById(client, id as string, context?.settings),
+    { enabled: !!isEditMode && Boolean(context?.settings) }
```

Comment:
`!!` is shorthand for `Boolean`, so can this be `Boolean(isEditMode) && Boolean(context?.settings)` instead for consistenacy?

Reply:
fixed as `{ enabled: isEditMode && Boolean(context?.settings) }`. The typecasting is already done above.